network-api/networkapi/templates/partials/primary_nav.html

```diff
@@ -33,12 +35,13 @@
               <div id="primary-nav-links">
                 <div class="d-flex align-items-center flex-wrap">
 
-                  {% block burger_and_logo %}
-                    <button class="burger {% if page.zen_nav is not True %} d-lg-none ml-md-0 {% endif %}" aria-label="{% trans 'Open menu' %}">
-                      <div class="burger-bar burger-bar-top"></div>
-                      <div class="burger-bar burger-bar-middle"></div>
-                      <div class="burger-bar burger-bar-bottom"></div>
-                    </button>
+                  <button class="burger {% if page.zen_nav is not True %} d-lg-none ml-md-0 {% endif %}" aria-label="{% trans 'Open menu' %}">
+                    <div class="burger-bar burger-bar-top"></div>
+                    <div class="burger-bar burger-bar-middle"></div>
+                    <div class="burger-bar burger-bar-bottom"></div>
```

Comment:
🍔 🍔 🍔  I am so hungry now

Reply:
that's why i made burger for breakfast this morning lol

Reply:
=D